GliaWidgets/Sources/Theme/Theme.Text.swift

```diff
@@ -0,0 +1,42 @@
+import UIKit
+
+extension Theme {
+    /// Text style.
```

Comment:
Because in this file we have descriptions as single words or very short phrases, lets skip the "." at the end.
```suggestion
    /// Text style
```

Reply:
fixed [2f6c133](https://github.com/salemove/ios-sdk-widgets/pull/887/commits/2f6c133f6a3b48578ec46a81c10e20675231e135)